### DISCUSSION TITLE
Issue in compiling Pan V1

### DISCUSSION THREAD

**Discussion by johnvprcam24573455623** [gtxaspec/wz_mini_hacks#586] [Development]
I am getting this error when running the command:
`./compile_image.sh pan`

```
root@debian-lenovo:/home/john/wz_mini_hacks/v2_install# ./compile_image.sh pan
wz_mini: this will download the latest firmware version from the vendor and compile a modified demo.bin.  Are you sure? [y/N]y
checking for tools
tools OK
create temporary work directory
check for local zip
local archive found
saving original firmware
removing temporary work directory
md5sum failed check, please supply a supported demo.zip file
```
I downloaded and renamed demo_4.10.9.1851.bin.zip to demo.zip

I added the file manually in v2_install folder:
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/88012789/8696c6e0-9f97-43c6-80cb-86bd8406b829)

the only thing that happens is that the demo.zip gets converted to demo_firmware-orig.zip
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/88012789/b3f43ed7-4fc1-4879-aac1-cde393b3ce32)

**Comment by johnvprcam24573455623** [comment]
looks like its expecting these two specific FW versions

```
if [[ $(md5sum ./v2_ro/demo*.zip) == *"a69b6d5ffdbce89463fa83f7f06ec10b"* ]]; then
	version_found="v2 4.9.8.1002"
elif [[ $(md5sum ./v2_ro/demo*.zip) == *"91793d32fd797a10df572f4f5d72bc55"* ]]; then
	version_found="pan v1 4.10.8.1002"
```

I thought that was working with also the lates FW 4.10.9.1851
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/88012789/51635a9a-e1c9-4511-86ea-2e9498131d09)
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/88012789/a7626a00-df2c-42c8-bfaa-986184506f26)

**Reply by BillCai06** [reply]
I run into the same problem

**Comment by johnvprcam24573455623** [comment]
trying with firmware 4.10.8.1002, howwere when then I want to connect the app I get " cannot find the specified network name"
